testcases/OpTestEEH.py

```diff
@@ -261,6 +309,7 @@ def runTest(self):
                     break
                 else:
                     print "#Waiting for PHB %s recovery. (%d/%d)" % (domain,i,tries)
+            self.gather_logs()
```

Comment:
I think a better solution is to grep out for errors/warnings in OPAL and kernel logs and fail the test if we get any.